[DOCS-2842] Update NodeJS log collection docs (#15468)
DataDog/documentation: content/en/logs/log_collection/nodejs.md

{{% /tab %}}
{{< /tabs >}}

> Check the content of the `<FILE_NAME>.log` file to confirm that Winston is logging in the JSON format:

Author: ```suggestion
Check the content of the `<FILE_NAME>.log` file to confirm that Winston is logging in JSON:
```

```json
{"level":"info","message":"Hello simple log!","timestamp":"2015-04-23T16:52:05.337Z"}
{"color":"blue","level":"info","message":"Hello log with metas","timestamp":"2015-04-23T16:52:05.339Z"}
```

## Configure your Datadog Agent

> Once [log collection is enabled][6], set up [custom log collection][7] to tail your log files and send them to Datadog.

Reviewer: ```suggestion
Once [log collection is enabled][6], set up [custom log collection][7] to tail your log files and send new logs to Datadog.
```

Reviewer: This one is up to you, I just remember having customers that were confused about why old logs were not being sent, so thought I'd add this in as a clarification

    sourcecategory: sourcecode
```

3. [Restart the Agent][9].

> 4. Run the [Agent’s status subcommand][10] and look for `java` under the `Checks` section to confirm logs are successfully submitted to Datadog.

Reviewer: Should this by `nodejs` or something similar instead of `java`?

Author: Yes, thanks for catching that!

> 4. Run the [Agent’s status subcommand][10] and look for `java` under the `Checks` section to confirm logs are successfully submitted to Datadog.

Author: ```suggestion
4. Run the [Agent’s status subcommand][10] and look for `nodejs` under the `Checks` section to confirm logs are successfully submitted to Datadog.
```

If logs are in JSON format, Datadog automatically [parses the log messages][11] to extract log attributes. Use the [Log Explorer][12] to view and troubleshoot your logs.

## Connect your service across logs and traces

If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
`env`, `service`, and `version` to your logs by [following the APM NodeJS instructions][3].

> **Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the agent configuration.

Reviewer: ```suggestion
**Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the Agent configuration.
```

## Agentless logging

> You can stream your logs from your application to Datadog without installing an Agent on your Host. However, it is recommended that you use an Agent to forward your logs as it provides a native connection management.

Reviewer: ```suggestion
You can stream your logs from your application to Datadog without installing an Agent on your host. However, it is recommended that you use an Agent to forward your logs as it provides a native connection management.
```
